digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetOperations.scala

```diff
@@ -310,12 +310,24 @@ trait PointAssetOperations {
     }
   }
 
+  def getPersistedAssetsByIdsWithExpire(ids: Set[Long]): Seq[PersistedAsset] = {
+    withDynSession {
+      getPersistedAssetsByIdsWithExpireWithoutTransaction(ids)
+    }
+  }
+
   def getPersistedAssetsByIdsWithoutTransaction(ids: Set[Long]): Seq[PersistedAsset] = {
       val idsStr = ids.toSeq.mkString(",")
       val filter = s"where a.asset_type_id = $typeId and a.id in ($idsStr)"
       fetchPointAssets(withFilter(filter))
   }
 
+  def getPersistedAssetsByIdsWithExpireWithoutTransaction(ids: Set[Long]): Seq[PersistedAsset] = {
+      val idsStr = ids.toSeq.mkString(",")
+      val filter = s"where a.asset_type_id = $typeId and a.id in ($idsStr)"
+    fetchPointAssetsWithExpired(withFilter(filter))
+  }
+
```

Comment:
some of that methods was created for trafficSignManager, please check if it continues to be used

Reply:
This method is used by getPersistedAssetsByIdsWithExpire mentioned above to be checked if still in use.
Since that method it's in use this method it's in use too.